runtime/ecl/org.eclipse.rcptt.tesla.ecl.impl/src/org/eclipse/rcptt/tesla/ecl/internal/impl/commands/SelectorService.java

```diff
@@ -127,7 +126,7 @@ public IStatus service(Command command, IProcess context)
 				path += "%" + index + "%";
 			}
 			if (column != null) {
-				path += "#" + column + "#";
+				path += "#@" + column + "#@";
```

Comment:
Lets move "#@" to some constant and make it even bigger value. "#@||*#" for example. Please check if this would work fine.